setup.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+import io
+import os
+import re
+import shutil
+import sys
+from setuptools import setup, find_packages
+
+
+def read(*names, **kwargs):
+    with io.open(
+        os.path.join(os.path.dirname(__file__), *names),
+        encoding=kwargs.get("encoding", "utf8")
+    ) as fp:
+        return fp.read()
+
+
+def find_version(*file_paths):
+    version_file = read(*file_paths)
+    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
+                              version_file, re.M)
+    if version_match:
+        return version_match.group(1)
+    raise RuntimeError("Unable to find version string.")
+
+
+VERSION = find_version('src', 'gluonnlp', '__init__.py')
+
+requirements = [
+    'numpy',
+    'spacy',
+    'sacremoses>=0.0.38',
+    'yacs>=0.1.6',
+    'sacrebleu',
+    'jieba',
+    'flake8',
+    'regex',
+    'scipy',
+    'h5py>=2.10'
+]
+
+setup(
+    # Metadata
+    name='gluonnlp',
+    version=VERSION,
+    python_requires='>=3.6',
```

Comment:
Some recent packages like Jax have dropped python3.5: https://github.com/google/jax. So I choose to also drop python3.5 @leezu @eric-haibin-lin @szha @szhengac .